samples/browser/src/main/java/org/mozilla/samples/browser/BrowserActivity.kt

```diff
@@ -73,11 +75,12 @@ open class BrowserActivity : AppCompatActivity(), ComponentCallbacks2 {
         when (name) {
             EngineView::class.java.name -> components.engine.createView(context, attrs).apply {
                 selectionActionDelegate = DefaultSelectionActionDelegate(
-                    components.store,
-                    context
-                ) {
-                    share(it)
-                }
+                    store = components.store,
+                    context = context,
+                    shareTextClicked = { share(it) },
+                    emailTextClicked = { email(it) },
+                    callTextClicked = { call(it) }
```

Comment:
Those can be the default values on `DefaultSelectionActionDelegate` as most of the people will do that :) wdyt?

Reply:
Done!